bio_auth_service/src/main/kotlin/fingerprint/FingerprintRoutes.kt

```diff
@@ -20,7 +19,6 @@ import org.kiva.bioauthservice.fingerprint.dtos.VerifyRequestDto
  */
 private object Paths {
     const val apiV1 = "/api/v1"
-    const val getPositions = "/positions/template"
```

Comment:
Expected?

Reply:
Yeah, I noticed it was an unused variable (and there was an unused import above too). I should probably tweak the lint settings to catch these things.